core/src/main/resources/error/error-classes.json

```diff
@@ -327,6 +327,78 @@
     ],
     "sqlState" : "42000"
   },
+  "INVALID_SUBQUERY_EXPRESSION" : {
+    "message" : [
+      "Invalid subquery:"
+    ],
+    "subClass" : {
+      "ACCESSING_OUTER_QUERY_COLUMN_IS_NOT_ALLOWED" : {
+        "message" : [
+          "Accessing outer query column is not allowed in this location<plan>"
```

Comment:
Yeah what we need here is the context, not the plan. Actually we need two locations. The place it's resolving to and the place it's resolving from.